### ISSUE TITLE
Support for Custom Attribute Naming with Non-camelCase Variables

### ISSUE THREAD

**Issue by LennartKleymann** [camunda-community-hub/zeebe-client-csharp-accelerator#5]
**Is your feature request related to a problem? Please describe.**

I am currently using the Zeebe.Client.Accelerator package for my C# project,. The package automatically deserializes the TInput object into an array of fetch variables. However, I've encountered an issue with variable naming conventions. While the[ official camunda documentation](https://docs.camunda.io/docs/components/concepts/variables/) allows the usage of camelCase, PascalCase, and snake_case for variable names, the Zeebe.Client.Accelerator package only supports camelCase. This limitation is causing problems in our integration with camunda, as we need the flexibility to use custom variable names that may not follow the camelCase convention.

**Describe the solution you'd like**

To address this limitation and enhance the package's functionality, I propose the implementation of a new custom attribute in C#. The custom attribute should be used to decorate C# properties and provide the necessary metadata for mapping these properties to the desired Zeebe variable names. With this attribute, users can specify the exact variable names they want to use, regardless of the variable naming convention.

**Expected Behavior:**

The package should support custom attribute mapping for properties to allow users to define custom variable names when deserializing TInput into the array of fetch variables. This custom attribute should enable us to map C# property names to custom Zeebe variable names in a straightforward manner.

**Example Usage:**
```csharp
public class MyWorkflowInput
{
    [ZeebeVariableName("OrderId")] // Using the custom attribute to map to Zeebe variable "OrderId"		
    public string OrderId { get; set; }

    [ZeebeVariableName("ItemList")] // Using the custom attribute to map to Zeebe variable "ItemList"
    public List<string> ItemList { get; set; }

    // Additional properties and their mappings as needed...
}
```

**Addional context**

existing mapping to fetchVariables: JobHandlerInfoProvider.GetFetchVariablesFromJobState(Type jobType) - Line: 171

**Comment by VonDerBeck** [comment]
Under the hood this project uses `System.Text.Json.JsonSerializer`. 

A compatible solution would be to use
```csharp
[JsonPropertyName("MY_Variable_name")]
public string MyVariable { get; set; }
```
This will indeed need a change in the code when detecting the variables to fetch, but all the rest should work out of box.

**Comment by VonDerBeck** [comment]
Hi @LennartKleymann,
I just released version 1.1.0 which supports JsonPropertyName and JsonIgnore attributes. This should help you to customize your attribute mapping.
